Resources/ArmTemplates/template.json

```diff
@@ -38,6 +44,9 @@
         "sharedAppServicePlanResourceGroup": {
             "type": "string",
             "defaultValue": ""
+        },
+        "JobProfileOverView": {
```

Comment:
I'd be tempted to either rename this to "JobProfileOverviewUrl" or to add a description saying what it should contain...  it's a bit vague as is

Reply:
Updated